CHANGELOG.md

```diff
@@ -10,7 +10,8 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 ### Added Features
 
 ### Removed Features
-- ROX-12316: As announced in 3.72, the permission `Cluster` replaces the deprecated permission `ClusterCVE`. 
+- ROX-12316: As announced in 3.72, the permission `Cluster` replaces the deprecated permission `ClusterCVE`.
+- ROX-13535: Built-in documentation link redirects now to the online version.
```

Comment:
Would you be so kind to mention that we also: don't ship the stand-alone docs container and don't embed docs in the product?